src/pages/resources/privacy-manifest.md

```diff
@@ -0,0 +1,40 @@
+Starting in Spring 2024, Apple will require developers to implement new privacy features to receive approval for apps submitted to the App Store. Some features are already in use, like Privacy Nutrition Labels. These labels help end-users understand the data collected by apps as well as third-party SDKs like those provided by Adobe. To facilitate this, Apple is requiring that app developers provide details in the Privacy Manifest regarding the data their apps (and included third-party SDKs) track.
+
+**Required Reasons API**
+
+To address concerns regarding fingerprinting, Apple will require developers to declare the reasons for using specific APIs. One of these APIs is the UserDefaults API, which Adobe currently uses to store user identifiers and lifecycle data. The Adobe Experience Platform Mobile SDK needs to store this data on the device in order to function, however, that data does not need to be stored in UserDefaults. Adobe has started using device storage rather than UserDefaults. In December, we released version 4.2.1 of the iOS Core extension that includes logic to migrate data stored by the AEP Mobile SDK away from UserDefaults to local storage for iOS and iPadOS.
+
+<InlineAlert variant="info" slots="text"/>
+
+tvOS does not allow local storage for applications, so until further notice, the AEP Mobile SDK will continue to store User Identifiers and Lifecycle Data in UserDefaults.
+
+**Specific Values to be included in the Privacy Manifest**
+
+AEP Mobile SDK customers are responsible for updating and maintaining their app’s Privacy Manifest. Which extensions customers use, and the specific actions performed using the AEP Mobile SDK, may affect what customers should include in the Privacy Manifest Data Use section. The below chart is intended to help guide customers’ understanding of how AEP Mobile SDK data can be used:
+
+| **AEP Mobile SDK Extension** | **Data** | [**Linked to User**](https://developer.apple.com/app-store/app-privacy-details/#linked-data) | [**Used For Tracking**](https://developer.apple.com/app-store/app-privacy-details/#user-tracking) | **Reason for Collection** |
+| --- | --- | --- | --- | --- |
+| <ul><li>Analytics</li><li>Core</li><li>Identity</li><li>Edge Identity</li></ul> | User ID | **No –** when customer is not capturing authenticated identities in the Identity Map<br/><br/>**Yes** **–** when customer is capturing authenticated identities in the Identity Map | **No –** when customer is not capturing authenticated identities in the Identity Map<br/><br/>**Yes** **–** when customer is capturing authenticated identities in the Identity Map | <ul><li>Developer’s Advertising or Marketing</li></ul> |
```

Comment:
For slightly better readability, I'd go with "when the customer is...".

This goes for all subsequent mentions of this as well.